src/lib.rs

```diff
@@ -92,12 +92,23 @@ pub struct LegalOfficerCase<AccountId, Hash, LocId, BlockNumber> {
 
 pub type LegalOfficerCaseOf<T> = LegalOfficerCase<<T as frame_system::Config>::AccountId, <T as pallet::Config>::Hash, <T as pallet::Config>::LocId, <T as frame_system::Config>::BlockNumber>;
 
+pub type LicenseItem = Vec<u8>;
+
+#[derive(Encode, Decode, Default, Clone, PartialEq, Eq, Debug, TypeInfo)]
+pub enum License<Hash> {
+	#[default]
+	None,
+	Specific(Hash),
+	Logion(Hash, Vec<LicenseItem>)
```

Comment:
Why not using directly a LOC ID here? The hash of the license is actually an item of the referenced LOC. Also, with regards to my comment above, I would use a more generic field for the "parameters" of the license (i.e. just a text field, so `Vec<u8>`).

Reply:
Agreed. Changed in 68c89bf1879e26f1746e24f25cfc59ce01c5d466